Source/Core/DolphinWX/Config/GeneralConfigPane.cpp

```diff
@@ -39,10 +39,9 @@ GeneralConfigPane::GeneralConfigPane(wxWindow* parent, wxWindowID id)
 
 void GeneralConfigPane::InitializeGUI()
 {
-	m_frame_limit_array_string.Add(_("Off"));
-	m_frame_limit_array_string.Add(_("Auto"));
-	for (int i = 5; i <= 120; i += 5) // from 5 to 120
-		m_frame_limit_array_string.Add(wxString::Format("%i", i));
+	m_throttler_array_string.Add(_("Disabled"));
```

Comment:
I agree. You might be able to disable a limit, but you can't disable a speed. I would prefer `Unlimited`.